media-fonts/joypixels/joypixels-6.5.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+FONT_SUFFIX="ttf"
+inherit font
+
+DESCRIPTION="joypixels font for better emoji and unicode support"
+HOMEPAGE="https://www.joypixels.com/"
+
```

Comment:
Remove empty line here and move KEYWORDS under SLOTS, see https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds#How_to_create_an_ebuild

Reply:
done